tests/tools/oecert/host/host.cpp

```diff
@@ -25,8 +28,10 @@ typedef struct _input_params
     const char* private_key_filename;
     const char* public_key_filename;
     const char* out_filename;
+    const char* endorsements_filename;
     bool gen_cert;
     bool gen_report;
+    bool gen_evidence;
```

Comment:
Avoid abbreviations like "gen".  Yes I see it was already that was in the other two lines so this is a non-blocking suggestion to expand it throughout this file.

Reply:
Sure, no problem!